compiler/src/dotty/tools/dotc/classpath/ClassPathFactory.scala

```diff
@@ -25,7 +25,8 @@ class ClassPathFactory {
     for {
       file <- expandPath(path, expandStar = false)
       dir <- Option(AbstractFile getDirectory file)
-    } yield createSourcePath(dir)
+    }
+    yield createSourcePath(dir)
```

Comment:
This is an unusual style, can the rewrite favor puttin the yield on the same line as the closing bracket if there is one ?

Reply:
The `yield` on the next line is consistent with how we format `else`. I think it's preferable if section keywords don't hide after a `}`.